voice-interface/src/main/java/com/derpgroup/derpwizard/voice/model/SsmlDocumentBuilder.java

```diff
@@ -82,6 +82,7 @@ public String toString() {
   private Set<String> ignoreTags;
   private List<List<StringBuilder>> paragraphs;
   private int index = 0;
+  private boolean conversationEnd = true;
```

Comment:
I'm sort of leaning that way, as well.  This code was done before CommonMetadata existed (I'm not actually sure why it's showing in this PR).  The question will be how to deal with serialization of that field.

Anyhow, I think that can be addressed with the later PR where the code is more settled.